### PR TITLE
[ASM] - Parent Playbook Input Typo

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-11327

## Description
AcceptedRiskDs is misspelled in "Cortex ASM - ASM Alert" as it should be AcceptedRiskIDs instead.

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , please review

**Comment by johnnywilkes** [comment]
@capanw / @BigEasyJ , please review

**Review by capanw — APPROVED** [review]

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_52.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex ASM - ASM Alert
+
+Fixed an issue with playbook input spelling.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_52.md:6]:
```suggestion
Fixed an issue with the spelling of a playbook input.
```

**Comment by ShirleyDenkberg** [comment]
@capanw @israelpoli @tomer-pan Doc review completed.

**Comment by johnnywilkes** [comment]
@tomer-pan / @israelpoli , this PR has been approved by Xpanse, please merge when possible

**Pull request merged**